Error when trying to install latest all the mods 10

### Describe the problem

After restarting the container to have the latest version of all the mods 10 installed, the update fails and the server does not start.

The `latest` image is being pulled before starting up the container.

I looked and did not see anyone else report this as of yet.

I tried with `CF_FORCE_SYNCHRONIZE=true` and `CF_FORCE_SYNCHRONIZE=false`. as well as `CF_FORCE_REINSTALL_MODLOADER=true` and `CF_FORCE_REINSTALL_MODLOADER=false`

I noticed in the logs that it is saying `More than one mod found with slug=`

> [!NOTE]
> Reverting / Pinning to the previous version (2.24) by using `CF_FILENAME_MATCHER` allows the system to successfully start, but my system already has that version installed.

### Container definition

``` yaml
networks:
  internal:
    driver: bridge
services:
  minecraft-curseforge:
    hostname: minecraft-curseforge
    container_name: minecraft-curseforge
    image: itzg/minecraft-server:java21-alpine
    networks:
    - internal
    ports: 
    - 25565:25565
    - 25575:25575
    restart: unless-stopped
    dns:
    - 192.168.2.7
    - 1.1.1.1
    - 1.0.0.1
    - 8.8.8.8
    - 8.8.4.4
    dns_search: bit13.local

    ulimits:
      nofile:
        soft: 65536
        hard: 65536
    deploy:
      resources:
        limits:
          memory: 24g
        reservations:
          memory: 24g
    environment:
      TYPE: AUTO_CURSEFORGE
      CF_API_KEY: '***************************************************************'
      CF_SLUG: all-the-mods-10
      VERSION: 1.21.1
      CF_FORCE_SYNCHRONIZE: true
      CF_EXCLUDE_MODS: 986380, 520110, 394468, 447673, 927064
      CF_FORCE_INCLUDE_MODS: ''
      CF_FORCE_REINSTALL_MODLOADER: true
      FETCH_TLS_HANDSHAKE_TIMEOUT: PT20S

      ENABLE_AUTOPAUSE: "TRUE"
      AUTOPAUSE_KNOCK_INTERFACE: 'eth0'

      SEED: '8483537205'
      VIEW_DISTANCE: 10

      ICON: "https://i.imgur.com/RV1PBPD.png"
      OVERRIDE_ICON: 'TRUE'

      OPS_FILE: 'http://andeddu.bit13.local:10070/tacobot/minecraft/ops.json'
      WHITELIST_FILE: 'http://andeddu.bit13.local:10070/tacobot/minecraft/whitelist.json'
      ENFORCE_WHITELIST: 'true'
      OVERRIDE_WHITELIST: 'true'

      REMOVE_OLD_MODS: 'false'

      LEVEL: taco_atm10-2

      MEMORY: 24G
      
      ENABLE_RCON: 'true'
      RCON_PASSWORD: '******************************'
      BROADCAST_RCON_TO_OPS: 'true'

      ALLOW_NETHER: 'true'
      ANNOUNCE_PLAYER_ACHIEVEMENTS: 'true'
      STOP_SERVER_ANNOUNCE_DELAY: 60
      STOP_DURATION: 60
      
      MOTD: The §l§6TACO§r §cMinecraft§r Server
      MAX_PLAYERS: 20
      SNOOPER_ENABLED: 'false'
      HARDCORE: 'false'
      DIFFICULTY: easy
      GENERATE_STRUCTURES: 'true'
      MAX_BUILD_HEIGHT: 256
      SPAWN_ANIMALS: 'true'
      SPAWN_NPCS: 'true'
      SPAWN_MONSTERS: 'true'
      SPAWN_PROTECTION: 0

      MODE: survival
      PVP: 'false'

      LEVEL_TYPE: amplified
      ALLOW_FLIGHT: 'true'
      ENABLE_ROLLING_LOGS: 'true'
      MAX_TICK_TIME: "-1"
      GUI: 'FALSE'
      UID: 1000
      GID: 1000
      TZ: 'America/Chicago'
      EULA: 'true'
      tty: 'true'
      stdin_open: 'true'

    volumes:
    - /etc/localtime:/etc/localtime:ro
    - /etc/timezone:/etc/timezone:ro
    - /mnt/container_data/minecraft/curseforge/modpacks:/modpacks
    - /mnt/container_data/minecraft/curseforge/data:/data
    - /mnt/data/Backup/minecraft/backups:/data/simplebackups/
    - /mnt/data/Backup/minecraft/downloads/atm10:/downloads

```


### Container logs

```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 24 1000 1000 4096 Jan 15 21:56 /data'
[init] Autopause functionality enabled
[mc-image-helper] 22:16:40.093 INFO  : Processing modpack 'All the Mods 10-2.25' (all-the-mods-10) @ 925200:6091559
[mc-image-helper] 22:16:40.347 ERROR : 'install-curseforge' command failed. Version is 1.40.10
me.itzg.helpers.errors.GenericException: More than one mod found with slug=
	at me.itzg.helpers.curseforge.CurseForgeApiClient.lambda$searchMod$2(CurseForgeApiClient.java:151)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.secondComplete(MonoFlatMap.java:245)
	at reactor.core.publisher.MonoFlatMap$FlatMapInner.onNext(MonoFlatMap.java:305)
	at reactor.core.publisher.FluxContextWrite$ContextWriteSubscriber.onNext(FluxContextWrite.java:107)
	at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.onNext(FluxDoFinally.java:113)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:158)
	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:104)
		at reactor.core.publisher.Mono.block(Mono.java:1779)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.resolveExcludeIncludes(CurseForgeInstaller.java:658)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.processModpack(CurseForgeInstaller.java:541)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.resolveModpackFileAndProcess(CurseForgeInstaller.java:395)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.installByRetrievingModpackZip(CurseForgeInstaller.java:257)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.lambda$install$3(CurseForgeInstaller.java:182)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:228)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:181)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:222)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:30)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2465)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2457)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2419)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2421)
		at picocli.CommandLine.execute(CommandLine.java:2174)
		at me.itzg.helpers.McImageHelper.main(McImageHelper.java:161)
[init] [ERROR] Failed to auto-install CurseForge modpack
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -50,7 +50,7 @@ RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
   --var version=${MC_SERVER_RUNNER_VERSION} --var app=mc-server-runner --file {{.app}} \
   --from ${GITHUB_BASEURL}/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
 
-ARG MC_HELPER_VERSION=1.40.10
+ARG MC_HELPER_VERSION=1.40.11
 ARG MC_HELPER_BASE_URL=${GITHUB_BASEURL}/itzg/mc-image-helper/releases/download/${MC_HELPER_VERSION}
 # used for cache busting local copy of mc-image-helper
 ARG MC_HELPER_REV=1

